example_analysis/0.configure_preprocess_params.ipynb

```diff
@@ -20,7 +20,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    SBS_REAL_IMAGES_FP.iterdir()
</pre><p>This also looks like testing code that can be removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cheeseman-lab/brieflow/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>